addons/payment_authorize/static/src/js/payment_form.js

```diff
@@ -0,0 +1,145 @@
+odoo.define('payment_authorize.payment_form', function (require) {
+"use strict";
+
+var ajax = require('web.ajax');
+var core = require('web.core');
+var PaymentForm = require('payment.payment_form');
+
+var _t = core._t;
+
+PaymentForm.include({
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * called when clicking on pay now or add payment event to create token for credit card/debit card.
+     *
+     * @private
+     * @param {Event} ev
+     * @param {DOMElement} checkedRadio
+     * @param {Boolean} addPmEvent
+     */
+    _createAuthorizeToken: function (ev, $checkedRadio, addPmEvent) {
+        var self = this;
+        var acquirerID = this.getAcquirerIdFromRadio($checkedRadio);
+        var acquirerForm = this.$('#o_payment_add_token_acq_' + acquirerID);
+        var inputsForm = $('input', acquirerForm);
+        var formData = self.getFormData(inputsForm);
+        if (this.options.partnerId === undefined) {
+            console.warn('payment_form: unset partner_id when adding new token; things could go wrong');
+        }
+        var AcceptJs = false;
+        if (formData.acquirer_environment === 'prod') {
+            AcceptJs = '//js.authorize.net/v3/AcceptUI.js';
```

Comment:
should be `ħttps` even if the current page is not, please specify the protocol in the url (same for next one)